src/_experimental.scss

```diff
@@ -158,4 +158,8 @@
 			color: #fff;
 		}
 	}
+
+	&.text-warning-light, .text-warning-light {
```

Comment:
Make it specific to the use case, and this color must pass Non-text contrast ratio.

```&.text-warning-light.glyphicon-warning-sign```